deploy.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+
+ENV=$1
+
+if [ -z "$ENV" ]; then
+    echo "ENV not set"
+    exit 1
+elif [ "$ENV" != "dev" ] && [ "$ENV" != "alpha" ] && [ "$ENV" != "staging" ] && [ "$ENV" != "prod" ] && [ "$ENV" != "perf" ]; then
+    echo "$ENV does not match supported environment"
+    exit 1
+fi
+
+BUCKET_NAME=`docker run -v $HOME:/root --rm broadinstitute/dsde-toolbox vault read -field=bucket_name "secret/dsde/firecloud/$ENV/avro-import/conf"`
+CLOUD_FUNCTION_NAME=`docker run -v $HOME:/root --rm broadinstitute/dsde-toolbox vault read -field=cloud_function_name "secret/dsde/firecloud/$ENV/avro-import/conf"`
+PROJECT=`docker run -v $HOME:/root --rm broadinstitute/dsde-toolbox vault read -field=project "secret/dsde/firecloud/$ENV/avro-import/conf"`
+ORCHESTRATION_SERVICE_ACCOUNT=`docker run -v $HOME:/root --rm broadinstitute/dsde-toolbox vault read -field=orchestration_service_account "secret/dsde/firecloud/$ENV/avro-import/conf"`
+RAWLS_SERVICE_ACCOUNT=`docker run -v $HOME:/root --rm broadinstitute/dsde-toolbox vault read -field=rawls_service_account "secret/dsde/firecloud/$ENV/avro-import/conf"`
+TOPIC_NAME=`docker run -v $HOME:/root --rm broadinstitute/dsde-toolbox vault read -field=topic_name "secret/dsde/firecloud/$ENV/avro-import/conf"`
+FUNCTION_DIRECTORY=function
+LIFECYCLE_RULE_FILE=lifecycle_rule.json
```

Comment:
would be nice to read `secret/dsde/firecloud/$ENV/avro-import/conf` once, then extract the various parts out of it. Not a blocker for this PR; can do this in a future optimization.

Reply:
Agreed this is not great